src/scripts/routes.js

```diff
@@ -0,0 +1,440 @@
+define([
+    "emby-button",
+    "emby-input",
+    "scripts/livetvcomponents",
+    "paper-icon-button-light",
+    "emby-itemscontainer",
+    "emby-collapse",
+    "emby-select",
+    "livetvcss",
+    "emby-checkbox",
+    "emby-slider",
+    "listViewStyle",
+    "dashboardcss"], function () {
+
+    function defineRoute(newRoute, dictionary) {
+        var baseRoute = Emby.Page.baseUrl();
+        var path = newRoute.path;
+        path = path.replace(baseRoute, "");
+        console.log("Defining route: " + path);
+        newRoute.dictionary = newRoute.dictionary || dictionary || "core";
+        Emby.Page.addRoute(path, newRoute);
+    }
+
+    console.log("Defining core routes");
+        defineRoute({
+            path: "/addplugin.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "addpluginpage"
+        });
+        defineRoute({
+            path: "/autoorganizelog.html",
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/channelsettings.html",
+            autoFocus: false,
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/addserver.html",
+            autoFocus: false,
+            anonymous: true,
+            startup: true,
+            controller: "addserver"
+        });
+        defineRoute({
+            path: "/dashboard.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "dashboardpage"
+        });
+        defineRoute({
+            path: "/dashboardgeneral.html",
+            controller: "dashboardgeneral",
+            autoFocus: false,
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/dashboardhosting.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "dashboardhosting"
+        });
+        defineRoute({
+            path: "/devices/devices.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "devices"
+        });
+        defineRoute({
+            path: "/devices/device.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "device"
+        });
+        defineRoute({
+            path: "/dlnaprofile.html",
+            autoFocus: false,
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/dlnaprofiles.html",
```

Comment:
is it intentional that there's `dlnaprofile` and plural `dlnaprofiles`?

Reply:
Don't have a clue, i just moved that code from site.js into routes.js for routes isolation.

Reply:
Can you add a `// TODO: investigate` or something?

Reply:
Looking into the code, dlnaprofiles is the page to list all dlna profiles in server settings. dlnaprofile is the detail page for a given dlna profile that you select. So i should say it's intentional and makes sense this way.